protocol/communities/community.go

```diff
@@ -1719,6 +1720,38 @@ func (o *Community) AddMember(publicKey *ecdsa.PublicKey, roles []protobuf.Commu
 	return changes, nil
 }
 
+func (o *Community) AddMemberToChat(chatID string, publicKey *ecdsa.PublicKey, roles []protobuf.CommunityMember_Roles) (*CommunityChanges, error) {
```

Comment:
It is not clear why we need it as it is used only in tests and even in tests we only check that this will not return an error, not check that` o *Community `was updated

Reply:
This is needed since channels can have different member list than the whole community due to channel permissions.

It is indirectly tested here: https://github.com/status-im/status-go/blob/af9eecd7afdb07fd258921a0478e16166cb9f2df/protocol/communities/community_encryption_key_action_test.go#L774-L778

It is used in tests only for now. I am building the utility that I will later use in the production.